merged upstream per https://github.com/numpy/SVML/pull/7 + https://github.com/numpy/numpy/pull/29481

From aec671eaa84135fddbdd35e6f759fbaedf814882 Mon Sep 17 00:00:00 2001
From: "Bernhard M. Wiedemann" <bwiedemann@suse.de>
Date: Wed, 30 Jul 2025 12:48:01 +0200
Subject: [PATCH] Add a .file entry to .s files

for reproducible builds.

Without this patch, building _multiarray_umath.cpython-312-x86_64-linux-gnu.so
with debuginfo contained the random temporary ccXXXXXX.o filenames used by gcc
for assembler output.

There are details in
https://bugzilla.opensuse.org/show_bug.cgi?id=1245203#c9
---

diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_d_ha.s
index 8f11c0e..72c2c3c 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_d_ha.s
@@ -14,6 +14,7 @@
 *******************************************************************************/
 
 
+	.file   "svml_z0_acos_d_ha.s"
 	.text
 .L_2__routine_start___svml_acos8_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_d_la.s
index 758149d..1e8b1d2 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_d_la.s
@@ -4,6 +4,7 @@
 *******************************************/
 
 
+	.file   "svml_z0_acos_d_la.s"
 	.text
 .L_2__routine_start___svml_acos8_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_h_la.s
index 9448d92..d8f91f8 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_h_la.s
@@ -13,6 +13,7 @@
  *
  */
 
+	.file   "svml_z0_acos_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_s_ha.s
index a9a8b22..2279b89 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_s_ha.s
@@ -14,6 +14,7 @@
 *******************************************************************************/
 
 
+	.file   "svml_z0_acos_s_ha.s"
 	.text
 .L_2__routine_start___svml_acosf16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_s_la.s
index c3b9d0c..74d1558 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acos_s_la.s
@@ -4,6 +4,7 @@
 *******************************************/
 
 
+	.file   "svml_z0_acos_s_la.s"
 	.text
 .L_2__routine_start___svml_acosf16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_d_ha.s
index 686074a..a09b0d3 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_d_ha.s
@@ -24,6 +24,7 @@
  */
 
 
+	.file   "svml_z0_acosh_d_ha.s"
 	.text
 .L_2__routine_start___svml_acosh8_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_d_la.s
index af0bf20..250b0fc 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_d_la.s
@@ -14,6 +14,7 @@
  */
 
 
+	.file   "svml_z0_acosh_d_la.s"
 	.text
 .L_2__routine_start___svml_acosh8_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_h_la.s
index 809a065..78db167 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_h_la.s
@@ -18,6 +18,7 @@
  *
  */
 
+	.file   "svml_z0_acosh_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_s_ha.s
index 8a431f0..785e949 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_s_ha.s
@@ -24,6 +24,7 @@
  */
 
 
+	.file   "svml_z0_acosh_s_ha.s"
 	.text
 .L_2__routine_start___svml_acoshf16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_s_la.s
index c12518c..eddf0f3 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_acosh_s_la.s
@@ -14,6 +14,7 @@
  */
 
 
+	.file   "svml_z0_acosh_s_la.s"
 	.text
 .L_2__routine_start___svml_acoshf16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_d_ha.s
index 6f85f65..2835079 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_d_ha.s
@@ -14,6 +14,7 @@
 *******************************************************************************/
 
 
+	.file   "svml_z0_asin_d_ha.s"
 	.text
 .L_2__routine_start___svml_asin8_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_d_la.s
index a84c68a..c1605c3 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_d_la.s
@@ -4,6 +4,7 @@
 *******************************************/
 
 
+	.file   "svml_z0_asin_d_la.s"
 	.text
 .L_2__routine_start___svml_asin8_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_h_la.s
index 1609b4f..d4e73ac 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_h_la.s
@@ -13,6 +13,7 @@
  *
  */
 
+	.file   "svml_z0_asin_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_s_ha.s
index a2a3894..231791a 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_s_ha.s
@@ -14,6 +14,7 @@
 *******************************************************************************/
 
 
+	.file   "svml_z0_asin_s_ha.s"
 	.text
 .L_2__routine_start___svml_asinf16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_s_la.s
index 07309c6..27f01ee 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asin_s_la.s
@@ -4,6 +4,7 @@
 *******************************************/
 
 
+	.file   "svml_z0_asin_s_la.s"
 	.text
 .L_2__routine_start___svml_asinf16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_d_ha.s
index cbb578e..ab6b868 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_d_ha.s
@@ -24,6 +24,7 @@
  */
 
 
+	.file   "svml_z0_asinh_d_ha.s"
 	.text
 .L_2__routine_start___svml_asinh8_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_d_la.s
index 79743d8..1bd76cd 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_d_la.s
@@ -14,6 +14,7 @@
  */
 
 
+	.file   "svml_z0_asinh_d_la.s"
 	.text
 .L_2__routine_start___svml_asinh8_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_h_la.s
index a135052..030dde8 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_h_la.s
@@ -12,6 +12,7 @@
  *
  */
 
+	.file   "svml_z0_asinh_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_s_ha.s
index 2a65ca5..5bff3dd 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_s_ha.s
@@ -24,6 +24,7 @@
  */
 
 
+	.file   "svml_z0_asinh_s_ha.s"
 	.text
 .L_2__routine_start___svml_asinhf16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_s_la.s
index 29df3e2..1ddf55c 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_asinh_s_la.s
@@ -14,6 +14,7 @@
  */
 
 
+	.file   "svml_z0_asinh_s_la.s"
 	.text
 .L_2__routine_start___svml_asinhf16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_d_ha.s
index 1221100..7c047fa 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_d_ha.s
@@ -26,6 +26,7 @@
  */
 
 
+	.file   "svml_z0_atan2_d_ha.s"
 	.text
 .L_2__routine_start___svml_atan28_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_d_la.s
index c2c1f54..bbc35ac 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_d_la.s
@@ -16,6 +16,7 @@
  */
 
 
+	.file   "svml_z0_atan2_d_la.s"
 	.text
 .L_2__routine_start___svml_atan28_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_h_la.s
index f0635b5..4189805 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_h_la.s
@@ -25,6 +25,7 @@
  *
  */
 
+	.file   "svml_z0_atan2_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_s_ha.s
index b8dd1ab..70b98fe 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_s_ha.s
@@ -26,6 +26,7 @@
  */
 
 
+	.file   "svml_z0_atan2_s_ha.s"
 	.text
 .L_2__routine_start___svml_atan2f16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_s_la.s
index ed77b56..4abefff 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan2_s_la.s
@@ -16,6 +16,7 @@
  */
 
 
+	.file   "svml_z0_atan2_s_la.s"
 	.text
 .L_2__routine_start___svml_atan2f16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_d_ha.s
index dc6e5e1..7e8f7e1 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_d_ha.s
@@ -26,6 +26,7 @@
  */
 
 
+	.file   "svml_z0_atan_d_ha.s"
 	.text
 .L_2__routine_start___svml_atan8_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_d_la.s
index 201060b..40875b9 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_d_la.s
@@ -16,6 +16,7 @@
  */
 
 
+	.file   "svml_z0_atan_d_la.s"
 	.text
 .L_2__routine_start___svml_atan8_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_h_la.s
index 5c80149..6c6ca08 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_h_la.s
@@ -15,6 +15,7 @@
  *
  */
 
+	.file   "svml_z0_atan_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_s_ha.s
index 4fa48cd..864151a 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_s_ha.s
@@ -26,6 +26,7 @@
  */
 
 
+	.file   "svml_z0_atan_s_ha.s"
 	.text
 .L_2__routine_start___svml_atanf16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_s_la.s
index 9868242..96412cf 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atan_s_la.s
@@ -16,6 +16,7 @@
  */
 
 
+	.file   "svml_z0_atan_s_la.s"
 	.text
 .L_2__routine_start___svml_atanf16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_d_ha.s
index f62a956..0810a24 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_d_ha.s
@@ -23,6 +23,7 @@
  */
 
 
+	.file   "svml_z0_atanh_d_ha.s"
 	.text
 .L_2__routine_start___svml_atanh8_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_d_la.s
index 63379da..75a584b 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_d_la.s
@@ -13,6 +13,7 @@
  */
 
 
+	.file   "svml_z0_atanh_d_la.s"
 	.text
 .L_2__routine_start___svml_atanh8_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_h_la.s
index f027c4d..215aa13 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_h_la.s
@@ -17,6 +17,7 @@
  *
  */
 
+	.file   "svml_z0_atanh_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_s_ha.s
index cea606c..3955a50 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_s_ha.s
@@ -23,6 +23,7 @@
  */
 
 
+	.file   "svml_z0_atanh_s_ha.s"
 	.text
 .L_2__routine_start___svml_atanhf16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_s_la.s
index 2f89983..6e6840d 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_atanh_s_la.s
@@ -13,6 +13,7 @@
  */
 
 
+	.file   "svml_z0_atanh_s_la.s"
 	.text
 .L_2__routine_start___svml_atanhf16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_d_ha.s
index e561446..eb1b725 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_d_ha.s
@@ -27,6 +27,7 @@
  */
 
 
+	.file   "svml_z0_cbrt_d_ha.s"
 	.text
 .L_2__routine_start___svml_cbrt8_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_d_la.s
index c557980..8200666 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_d_la.s
@@ -17,6 +17,7 @@
  */
 
 
+	.file   "svml_z0_cbrt_d_la.s"
 	.text
 .L_2__routine_start___svml_cbrt8_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_h_la.s
index 86b4a70..64192dc 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_h_la.s
@@ -16,6 +16,7 @@
  *
  */
 
+	.file   "svml_z0_cbrt_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_s_ha.s
index 50b5794..fc22b44 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_s_ha.s
@@ -27,6 +27,7 @@
  */
 
 
+	.file   "svml_z0_cbrt_s_ha.s"
 	.text
 .L_2__routine_start___svml_cbrtf16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_s_la.s
index b15e2f0..7c7c5a2 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cbrt_s_la.s
@@ -17,6 +17,7 @@
  */
 
 
+	.file   "svml_z0_cbrt_s_la.s"
 	.text
 .L_2__routine_start___svml_cbrtf16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_d_ha.s
index 7ab1561..a9f1f54 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_d_ha.s
@@ -39,6 +39,7 @@
  */
 
 
+	.file   "svml_z0_cos_d_ha.s"
 	.text
 .L_2__routine_start___svml_cos8_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_d_la.s
index 043e3ce..e87843d 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_d_la.s
@@ -18,6 +18,7 @@
  */
 
 
+	.file   "svml_z0_cos_d_la.s"
 	.text
 .L_2__routine_start___svml_cos8_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_h_la.s
index 48bc597..a0e9be4 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_h_la.s
@@ -27,6 +27,7 @@
  *
  */
 
+	.file   "svml_z0_cos_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_s_ha.s
index 5ab48a9..82f8319 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_s_ha.s
@@ -38,6 +38,7 @@
  */
 
 
+	.file   "svml_z0_cos_s_ha.s"
 	.text
 .L_2__routine_start___svml_cosf16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_s_la.s
index e1db29e..1a0542c 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cos_s_la.s
@@ -28,6 +28,7 @@
  */
 
 
+	.file   "svml_z0_cos_s_la.s"
 	.text
 .L_2__routine_start___svml_cosf16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_d_ha.s
index a82bdc6..4c9f0f2 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_d_ha.s
@@ -14,6 +14,7 @@
 *******************************************************************************/
 
 
+	.file   "svml_z0_cosh_d_ha.s"
 	.text
 .L_2__routine_start___svml_cosh8_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_d_la.s
index d8ef312..b91b93a 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_d_la.s
@@ -4,6 +4,7 @@
 *******************************************/
 
 
+	.file   "svml_z0_cosh_d_la.s"
 	.text
 .L_2__routine_start___svml_cosh8_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_h_la.s
index 0c66121..e01323c 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_h_la.s
@@ -19,6 +19,7 @@
  *
  */
 
+	.file   "svml_z0_cosh_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_s_ha.s
index f49a7f0..1a34dc8 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_s_ha.s
@@ -14,6 +14,7 @@
 *******************************************************************************/
 
 
+	.file   "svml_z0_cosh_s_ha.s"
 	.text
 .L_2__routine_start___svml_coshf16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_s_la.s
index 58b5ec2..0f89c9e 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_cosh_s_la.s
@@ -4,6 +4,7 @@
 *******************************************/
 
 
+	.file   "svml_z0_cosh_s_la.s"
 	.text
 .L_2__routine_start___svml_coshf16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp10_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp10_h_la.s
index df38cb1..4bd0f27 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp10_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp10_h_la.s
@@ -25,6 +25,7 @@
  *
  */
 
+	.file   "svml_z0_exp10_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_d_ha.s
index e6b758c..0c35863 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_d_ha.s
@@ -14,6 +14,7 @@
 *******************************************************************************/
 
 
+	.file   "svml_z0_exp2_d_ha.s"
 	.text
 .L_2__routine_start___svml_exp28_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_d_la.s
index 8806509..8f5c4f6 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_d_la.s
@@ -4,6 +4,7 @@
 *******************************************/
 
 
+	.file   "svml_z0_exp2_d_la.s"
 	.text
 .L_2__routine_start___svml_exp28_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_h_la.s
index 00e4921..3bbdc93 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_h_la.s
@@ -34,6 +34,7 @@
  *
  */
 
+	.file   "svml_z0_exp2_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_s_ha.s
index 15641ac..c5ebe4d 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_s_ha.s
@@ -14,6 +14,7 @@
 *******************************************************************************/
 
 
+	.file   "svml_z0_exp2_s_ha.s"
 	.text
 .L_2__routine_start___svml_exp2f16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_s_la.s
index a57e0f0..e169145 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp2_s_la.s
@@ -4,6 +4,7 @@
 *******************************************/
 
 
+	.file   "svml_z0_exp2_s_la.s"
 	.text
 .L_2__routine_start___svml_exp2f16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_d_ha.s
index 0a589e9..55ed611 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_d_ha.s
@@ -36,6 +36,7 @@
  */
 
 
+	.file   "svml_z0_exp_d_ha.s"
 	.text
 .L_2__routine_start___svml_exp8_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_d_la.s
index 54b2b2c..7de46b6 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_d_la.s
@@ -26,6 +26,7 @@
  */
 
 
+	.file   "svml_z0_exp_d_la.s"
 	.text
 .L_2__routine_start___svml_exp8_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_h_la.s
index 2cb9fc7..274898f 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_h_la.s
@@ -16,6 +16,7 @@
  *
  */
 
+	.file   "svml_z0_exp_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_s_ha.s
index 84e5e1d..fded195 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_s_ha.s
@@ -37,6 +37,7 @@
  */
 
 
+	.file   "svml_z0_exp_s_ha.s"
 	.text
 .L_2__routine_start___svml_expf16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_s_la.s
index 1859464..41e2172 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_exp_s_la.s
@@ -27,6 +27,7 @@
  */
 
 
+	.file   "svml_z0_exp_s_la.s"
 	.text
 .L_2__routine_start___svml_expf16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_d_ha.s
index a57ec13..f0514fe 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_d_ha.s
@@ -27,6 +27,7 @@
  */
 
 
+	.file   "svml_z0_expm1_d_ha.s"
 	.text
 .L_2__routine_start___svml_expm18_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_d_la.s
index 1362268..d65c2f0 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_d_la.s
@@ -17,6 +17,7 @@
  */
 
 
+	.file   "svml_z0_expm1_d_la.s"
 	.text
 .L_2__routine_start___svml_expm18_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_h_la.s
index f459d88..9a707f1 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_h_la.s
@@ -17,6 +17,7 @@
  *
  */
 
+	.file   "svml_z0_expm1_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_s_ha.s
index 8abae95..b987b43 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_s_ha.s
@@ -27,6 +27,7 @@
  */
 
 
+	.file   "svml_z0_expm1_s_ha.s"
 	.text
 .L_2__routine_start___svml_expm1f16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_s_la.s
index d64cf61..81fe691 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_expm1_s_la.s
@@ -17,6 +17,7 @@
  */
 
 
+	.file   "svml_z0_expm1_s_la.s"
 	.text
 .L_2__routine_start___svml_expm1f16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_d_ha.s
index bfd0fd6..bcbed5f 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_d_ha.s
@@ -26,6 +26,7 @@
  */
 
 
+	.file   "svml_z0_log10_d_ha.s"
 	.text
 .L_2__routine_start___svml_log108_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_d_la.s
index 4c97467..a7cfc92 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_d_la.s
@@ -16,6 +16,7 @@
  */
 
 
+	.file   "svml_z0_log10_d_la.s"
 	.text
 .L_2__routine_start___svml_log108_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_h_la.s
index 37bfaef..3f51bee 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_h_la.s
@@ -15,6 +15,7 @@
  *
  */
 
+	.file   "svml_z0_log10_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_s_ha.s
index bc2e2d5..6e8c3a6 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_s_ha.s
@@ -28,6 +28,7 @@
  */
 
 
+	.file   "svml_z0_log10_s_ha.s"
 	.text
 .L_2__routine_start___svml_log10f16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_s_la.s
index c824675..8c95cef 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log10_s_la.s
@@ -18,6 +18,7 @@
  */
 
 
+	.file   "svml_z0_log10_s_la.s"
 	.text
 .L_2__routine_start___svml_log10f16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_d_ha.s
index 28ade0f..1187a72 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_d_ha.s
@@ -28,6 +28,7 @@
  */
 
 
+	.file   "svml_z0_log1p_d_ha.s"
 	.text
 .L_2__routine_start___svml_log1p8_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_d_la.s
index d1cf448..2743531 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_d_la.s
@@ -18,6 +18,7 @@
  */
 
 
+	.file   "svml_z0_log1p_d_la.s"
 	.text
 .L_2__routine_start___svml_log1p8_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_h_la.s
index 8189230..590987a 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_h_la.s
@@ -15,6 +15,7 @@
  *
  */
 
+	.file   "svml_z0_log1p_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_s_ha.s
index 25cf31a..2be8614 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_s_ha.s
@@ -28,6 +28,7 @@
  */
 
 
+	.file   "svml_z0_log1p_s_ha.s"
 	.text
 .L_2__routine_start___svml_log1pf16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_s_la.s
index ed7a6af..19e1e19 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log1p_s_la.s
@@ -4,6 +4,7 @@
 *******************************************/
 
 
+	.file   "svml_z0_log1p_s_la.s"
 	.text
 .L_2__routine_start___svml_log1pf16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_d_ha.s
index 602a963..49117ef 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_d_ha.s
@@ -26,6 +26,7 @@
  */
 
 
+	.file   "svml_z0_log2_d_ha.s"
 	.text
 .L_2__routine_start___svml_log28_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_d_la.s
index 53e9c1c..87bb0a9 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_d_la.s
@@ -16,6 +16,7 @@
  */
 
 
+	.file   "svml_z0_log2_d_la.s"
 	.text
 .L_2__routine_start___svml_log28_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_h_la.s
index d7e776c..881e8e8 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_h_la.s
@@ -15,6 +15,7 @@
  *
  */
 
+	.file   "svml_z0_log2_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_s_ha.s
index 1f58f79..21392e6 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_s_ha.s
@@ -28,6 +28,7 @@
  */
 
 
+	.file   "svml_z0_log2_s_ha.s"
 	.text
 .L_2__routine_start___svml_log2f16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_s_la.s
index ac713d8..66d90f0 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log2_s_la.s
@@ -18,6 +18,7 @@
  */
 
 
+	.file   "svml_z0_log2_s_la.s"
 	.text
 .L_2__routine_start___svml_log2f16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_d_ha.s
index a7b6212..8dff30a 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_d_ha.s
@@ -27,6 +27,7 @@
  */
 
 
+	.file   "svml_z0_log_d_ha.s"
 	.text
 .L_2__routine_start___svml_log8_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_d_la.s
index 2d20084..4ff3939 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_d_la.s
@@ -17,6 +17,7 @@
  */
 
 
+	.file   "svml_z0_log_d_la.s"
 	.text
 .L_2__routine_start___svml_log8_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_h_la.s
index 0972f2b..e10f87e 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_h_la.s
@@ -15,6 +15,7 @@
  *
  */
 
+	.file   "svml_z0_log_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_s_ha.s
index 05695a7..df13904 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_s_ha.s
@@ -29,6 +29,7 @@
  */
 
 
+	.file   "svml_z0_log_s_ha.s"
 	.text
 .L_2__routine_start___svml_logf16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_s_la.s
index b399950..20683db 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_log_s_la.s
@@ -19,6 +19,7 @@
  */
 
 
+	.file   "svml_z0_log_s_la.s"
 	.text
 .L_2__routine_start___svml_logf16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_d_ha.s
index 69d887b..4834c07 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_d_ha.s
@@ -87,6 +87,7 @@
  */
 
 
+	.file   "svml_z0_pow_d_ha.s"
 	.text
 .L_2__routine_start___svml_pow8_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_d_la.s
index 774942a..b331ae3 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_d_la.s
@@ -77,6 +77,7 @@
  */
 
 
+	.file   "svml_z0_pow_d_la.s"
 	.text
 .L_2__routine_start___svml_pow8_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_h_la.s
index ecefbc4..9b5fd9b 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_h_la.s
@@ -16,6 +16,7 @@
  *
  */
 
+	.file   "svml_z0_pow_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_s_ha.s
index 5b706bb..19b6851 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_s_ha.s
@@ -28,6 +28,7 @@
  */
 
 
+	.file   "svml_z0_pow_s_ha.s"
 	.text
 .L_2__routine_start___svml_powf16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_s_la.s
index 31900b8..2e47192 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_pow_s_la.s
@@ -18,6 +18,7 @@
  */
 
 
+	.file   "svml_z0_pow_s_la.s"
 	.text
 .L_2__routine_start___svml_powf16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_d_ha.s
index 027c694..687bf89 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_d_ha.s
@@ -37,6 +37,7 @@
  */
 
 
+	.file   "svml_z0_sin_d_ha.s"
 	.text
 .L_2__routine_start___svml_sin8_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_d_la.s
index 8c7f5a6..3497c76 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_d_la.s
@@ -18,6 +18,7 @@
  */
 
 
+	.file   "svml_z0_sin_d_la.s"
 	.text
 .L_2__routine_start___svml_sin8_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_h_la.s
index d43bb22..356a751 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_h_la.s
@@ -16,6 +16,7 @@
  *
  */
 
+	.file   "svml_z0_sin_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_s_ha.s
index 950dfd1..283371d 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_s_ha.s
@@ -39,6 +39,7 @@
  */
 
 
+	.file   "svml_z0_sin_s_ha.s"
 	.text
 .L_2__routine_start___svml_sinf16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_s_la.s
index 05103f5..f813361 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sin_s_la.s
@@ -29,6 +29,7 @@
  */
 
 
+	.file   "svml_z0_sin_s_la.s"
 	.text
 .L_2__routine_start___svml_sinf16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_d_ha.s
index 7eff7f6..28bbf80 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_d_ha.s
@@ -14,6 +14,7 @@
 *******************************************************************************/
 
 
+	.file   "svml_z0_sinh_d_ha.s"
 	.text
 .L_2__routine_start___svml_sinh8_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_d_la.s
index 70ea648..d0ce5c4 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_d_la.s
@@ -4,6 +4,7 @@
 *******************************************/
 
 
+	.file   "svml_z0_sinh_d_la.s"
 	.text
 .L_2__routine_start___svml_sinh8_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_h_la.s
index fb56a38..2ff62fc 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_h_la.s
@@ -19,6 +19,7 @@
  *
  */
 
+	.file   "svml_z0_sinh_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_s_ha.s
index 27b1612..5fad91b 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_s_ha.s
@@ -14,6 +14,7 @@
 *******************************************************************************/
 
 
+	.file   "svml_z0_sinh_s_ha.s"
 	.text
 .L_2__routine_start___svml_sinhf16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_s_la.s
index 11893ce..fb2f9e5 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_sinh_s_la.s
@@ -4,6 +4,7 @@
 *******************************************/
 
 
+	.file   "svml_z0_sinh_s_la.s"
 	.text
 .L_2__routine_start___svml_sinhf16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_d_ha.s
index aeb5733..6742511 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_d_ha.s
@@ -30,6 +30,7 @@
  */
 
 
+	.file   "svml_z0_tan_d_ha.s"
 	.text
 .L_2__routine_start___svml_tan8_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_d_la.s
index e1cea7b..0b1b940 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_d_la.s
@@ -20,6 +20,7 @@
  */
 
 
+	.file   "svml_z0_tan_d_la.s"
 	.text
 .L_2__routine_start___svml_tan8_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_h_la.s
index 52bee9a..6079df2 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_h_la.s
@@ -16,6 +16,7 @@
  *
  */
 
+	.file   "svml_z0_tan_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_s_ha.s
index a0d87fa..fab8731 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_s_ha.s
@@ -30,6 +30,7 @@
  */
 
 
+	.file   "svml_z0_tan_s_ha.s"
 	.text
 .L_2__routine_start___svml_tanf16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_s_la.s
index d154702..cf57cf4 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tan_s_la.s
@@ -20,6 +20,7 @@
  */
 
 
+	.file   "svml_z0_tan_s_la.s"
 	.text
 .L_2__routine_start___svml_tanf16_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_d_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_d_ha.s
index 6037979..be78823 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_d_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_d_ha.s
@@ -69,6 +69,7 @@
  */
 
 
+	.file   "svml_z0_tanh_d_ha.s"
 	.text
 .L_2__routine_start___svml_tanh8_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_d_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_d_la.s
index cd0675c..cbdb585 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_d_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_d_la.s
@@ -59,6 +59,7 @@
  */
 
 
+	.file   "svml_z0_tanh_d_la.s"
 	.text
 .L_2__routine_start___svml_tanh8_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_h_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_h_la.s
index e075059..cfedd0e 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_h_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_h_la.s
@@ -26,6 +26,7 @@
  *
  */
 
+	.file   "svml_z0_tanh_h_la.s"
         .text
 
         .align    16,0x90
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_s_ha.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_s_ha.s
index 6b73844..8dd5a10 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_s_ha.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_s_ha.s
@@ -69,6 +69,7 @@
  */
 
 
+	.file   "svml_z0_tanh_s_ha.s"
 	.text
 .L_2__routine_start___svml_tanhf16_ha_z0_0:
 
diff --git a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_s_la.s b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_s_la.s
index 48b48a5..a2c1556 100644
--- a/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_s_la.s
+++ b/numpy/_core/src/umath/svml/linux/avx512/svml_z0_tanh_s_la.s
@@ -59,6 +59,7 @@
  */
 
 
+	.file   "svml_z0_tanh_s_la.s"
 	.text
 .L_2__routine_start___svml_tanhf16_z0_0:
 
